tests/middleware/test_session_middleware.py

```diff
@@ -20,25 +20,16 @@
 )
 from starlite.testing import create_test_client
 
-if TYPE_CHECKING:
-    from starlette.types import Receive, Scope, Send
-
-TEST_SECRET = SecretBytes(urandom(16))
-
-
-async def mock_asgi_app(scope: "Scope", receive: "Receive", send: "Send") -> None:
-    pass
-
 
 @pytest.mark.parametrize(
     "secret, should_raise",
     [
-        [urandom(16), False],
-        [urandom(24), False],
-        [urandom(32), False],
-        [urandom(17), True],
-        [urandom(4), True],
-        [urandom(100), True],
+        [os.urandom(16), False],
```

Comment:
Whats the reasoning for using `os.urandom` rather than `urandom`?

Reply:
Function imports from the standard library, keep their namespace implicit to their base module name.